opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -555,10 +571,7 @@ def test_aaf_nesting(self):
         )
         self.assertEqual(
             otio.opentime.TimeRange(
-                start_time=otio.opentime.RationalTime(32, 24),
-                # TODO: should actually be this, but we're not getting the
-                # media timecode offset correctly from the AAF...
-                # start_time=otio.opentime.RationalTime(86432, 24),
+                start_time=otio.opentime.RationalTime(86432, 24),
```

Comment:
I liked the way you wrote `86400+32` in the other cases, it makes it clear where this number came from.